mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/MappingRegistry.java

```diff
@@ -1,15 +1,60 @@
 package com.interface21.webmvc.servlet.mvc.tobe;
 
+import com.interface21.context.stereotype.Controller;
+import com.interface21.web.bind.annotation.RequestMapping;
 import com.interface21.web.bind.annotation.RequestMethod;
 
 import java.lang.reflect.Method;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.Collectors;
 
 class MappingRegistry {
     private final Map<RequestMappingInfo, HandlerExecution> handlerExecutions = new HashMap<>();
 
-    public void register(final RequestMappingInfo mapping, final Object handler, final Method method) {
+    public void registerControllers(final List<Object> controllers) {
+        controllers.forEach(this::detectHandlerMethods);
+    }
+
+    private void detectHandlerMethods(final Object handler) {
+        Class<?> clazz = handler.getClass();
+        Method[] handlerMethods = clazz.getDeclaredMethods();
+
+        StringBuilder parentPath = extractClassLevelUrlPath(clazz);
+
+        final String basePath = parentPath.toString();
+
+        final Map<Method, RequestMappingInfo> methods = extractMethodMappings(handlerMethods, basePath);
```

Comment:
```suggestion
        final String basePath = parentPath.toString();
        final Map<Method, RequestMappingInfo> methods = extractMethodMappings(handlerMethods, basePath);
```
어떠한 경우에 띄어쓰기를 하면 좋을까요?